tests/layer_tests/common/utils/tf_utils.py

```diff
@@ -162,8 +162,8 @@ def transpose_nchw_to_nhwc(data, use_new_frontend, use_old_api):
         return data
 
 
-def transpose_nhwc_to_nchw(data, use_new_frontend, use_old_api):
-    if use_new_frontend or not use_old_api:
+def transpose_nhwc_to_nchw(data, use_new_frontend):
+    if use_new_frontend:
```

Comment:
Is `use_new_frontend` still needed?

Reply:
it is a different story, we can remove this in separate PR and it does not block removing API 1.0